docs/index.rst

```diff
@@ -20,100 +16,220 @@ quickly be imported in your custom chain, as well as a
 simple framework for adding the custom functionality unique
 to your project.
 
-Blockchain Basics
------------------
-
-Some background material to help you get oriented with the
-concepts behind blockchains in general and tendermint/weave
-in particular. It is quite helpful to have a basic
-understanding of these concepts before trying to build on weave.
+Some of the highlights of Weave include a Merkle-tree backed data store, 
+a highly configurable extension system that also applies to the core logic such
+as fees and signature validation. Weave also brings powerful customizations 
+initialised from the genesis file. In addition there is a simple ORM 
+which sits on top of a key-value store that also has proveable secondary indexes.
+There is a flexible permissioning system to use contracts as first-class actors, 
+“No empty blocks” for quick synchronizing on quiet chains, 
+and the ability to introduce “product fees” for transactions that need to
+charge more than the basic anti-spam fees. We recently added support for
```

Comment:
I think it's best to avoid words that rely on relative point in time, such as "recently". Otherwise, we must revisit it in some weeks.